<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nuo</class>
 <widget class="QWidget" name="Nuo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuo</string>
  </property>
  <widget class="QPushButton" name="zbiorKart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Dobierz Karte</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>551</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="talia1"/>
  </widget>
  <widget class="QPushButton" name="logowanie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zaloguj Sie</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="sterta"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="talia2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
